client/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@ahmedisam99  Is this file gives us a value for the code?

Reply:
Yes, of course, we will yet change it.
check this: https://stackoverflow.com/questions/45186993/what-is-public-manifest-json-file-in-create-react-app

Reply:
Thanks ^_^